### PR TITLE
remove obsoleted stuff and fix builds@RHEL7

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

**Comment by xsuchy** [comment]
retest this please

**Comment by xsuchy** [comment]
retest this please

**Comment by mkutlak** [comment]
retest this please

File: faf.spec.in [hunk 1]
```diff
@@ -10,37 +10,30 @@ Name: faf
...
 
 BuildArch: noarch
 
-%if 0%{?rhel:%{rhel} < 7}
-%define sqlalchemy python-sqlalchemy0.8
-%define python_argparse python-argparse
-%define satyr_dep satyr-python >= 0.16
-%else
 %define sqlalchemy python-sqlalchemy >= 0.8
 %define python_argparse python
```
Comment (marusak) [inline faf.spec.in]: this is useless

**Comment by marusak** [comment]
Closed by https://github.com/abrt/faf/commit/b82fadc6dbe2442b2d0a3b52690c94a52fefe57e

**Pull request closed without merging**